meeting-notes/20190108.md

```diff
@@ -0,0 +1,175 @@
+   - Started contributing to ipfs-log:
+     - Use IPLD: https://github.com/orbitdb/ipfs-log/pull/213
+     - Helped review other PRs related to improvements we discussed with ipfs-log developers
+ - In progress:
+     - Add timeouts to ipfs-log read & write functions: https://github.com/orbitdb/ipfs-log/issues/199
+     - Discussing the "Collaboration Lifecycle" with Marco Oliveira and Pedro Santos: https://github.com/peer-base/peer-base/issues/227
+ - Next:
+     - Keep working on the Identity breakdown, assign people to high-level points outlined there and start working!
+
+#### @andreforsousa
+ - Concluded:
+   - Draft proposal for IDM sitemap with the priorities regarding user-journeys & pages
+   - Helped @satazor with the gist for IDM breakdown
+ - In progress:
+   - Peer-base: Collaboration Lifecycle and UI/UX guidelines with @satazor and @marcooliveira (already uploaded to GitHub)
+   - Kickstart (yesterday) new concept for "IDM"
+   - Follow the current implementation on Discussify animations
+ - Blocked:
+ - Next:
+   - Multiple approaches/concepts for "IDM" UI
+
+#### @dirkmc
+ - Concluded:
+ - In progress:
+   - Fixes in libp2p-switch
+ - Blocked:
+ - Next:
+   - Some changes to how connection management works
+
+#### @marcooliveira
+- Concluded:
+    - Breakdown of peer-base plan (mainly Strategic Product Management Activities and Outbound Activities)
+    - 2019 OKRs
+- In progress:
+    - Porto ProtoSchool chapter
+    - Collaboration Lifecycle draft (current status at https://gist.github.com/marcooliveira/fa5c31aded5b69c9c7deff5b73351729)
+        - Need to discuss with @satazor
+        - Need to discuss with @pgte
+    - Start discussing new iteration on IDM UX & UI with André Cruz & Sousa
+- Blocked:
+- Next:
+    - Events dapp pitch
+    - Define more specific plan for peer-base
+        - resources
+        - milestones
+    - Get approval for:
+        - peer-base plan
+        - Event dapp
+    - Get clarity on what's going to happen with Porto ProtoSchool
+        - Talk to Teri
+        - Talk to Jim Pick
+
+ -------------
+
+## Notes:
+- Jim demoed a peer-base RGA CRDT working in a terminal using nodejs. XState was used as state machine to change between starting, sending, saved, receiving, etc..
+- Pedro mentioned we must rename peer-base because the owner of the peerbase name declined to give it to us. He will make a pool with a few suggestions so that we can decide on the new name.
+- Pedro showed a PR about the WG roadmap in which we all participated in response to Dirk.
+- Arkady tried to reach a 3rd party to adopt our tech but it's too soon. We need to make the stack more stable and reliable first.
+- Adin and Pedro mentioned that we should learn from what textile and bring that knowledge to peer-base. Later on, and if textile agrees, they can replace parts that they built with parts from peer-base.
```

Comment:
Or the other way around :)

Reply:
True

Reply:
I don't necessarily agree that they're going to use parts of peer-base. They're more advanced than peer-base in many aspects. Right now we're focusing on a) learning what they did and b) their difficulties and how we can help.